linkerd/core/src/main/scala/io/buoyant/linkerd/Server.scala

```diff
@@ -119,12 +121,16 @@ class ServerConfig { config =>
       case Some(ps) => ApplicationProtocols.Supported(ps)
       case None => ApplicationProtocols.Unspecified
     }
-    Transport.ServerSsl(Some(SslServerConfiguration(
+    // The deprecated LegacyKeyServerEngineFactory allows us to accept PKCS#1 formatted keys.
+    // We should remove this and replace it with Netty4ServerEngineFactory once we no longer allow
+    // PKCS#1 keys.
+    @silent val sslServerEngine = SslServerEngineFactory.Param(LegacyKeyServerEngineFactory)
```

Comment:
Not easily, I don't think.  We'd need to open up the key ourselves and figure out the format.